src/parser/jobs/drg/Weaving.js

```diff
@@ -0,0 +1,21 @@
+import CoreWeaving from 'parser/core/modules/Weaving'
+import ACTIONS from 'data/ACTIONS'
+
+const JUMPS = [
+	ACTIONS.JUMP.id,
+	ACTIONS.SPINESHATTER_DIVE.id,
+	ACTIONS.DRAGONFIRE_DIVE.id,
+]
+
+export default class Weaving extends CoreWeaving {
+	isBadWeave(weave/*, maxWeaves*/) {
+		for (let i = 0; i < weave.weaves.length; i++) {
```

Comment:
Could probably be simplified w/ a `weaves.some(...)`, but it'll do the trick 👍

Reply:
True, I may clean that up later in a maintenance PR.